lxqtbacklight/linux_backend/driver/libbacklight_backend.c

```diff
@@ -178,12 +175,13 @@ char *lxqt_backlight_backend_get_driver()
     char *driver;
     int n;
     char type[1024];
+    const char *sysfs_backlight_dir = "/sys/class/backlight";
```

Comment:
How about `static`?

Reply:
Yes good hint. I refactored to make this a global static constant. Also added another commit to avoid duplicate directory specification in arguments to `open_driver_file()`.